lte/gateway/c/core/oai/tasks/amf/amf_app_ue_context.cpp

```diff
@@ -785,13 +800,29 @@ void amf_ue_context_update_ue_sig_connection_state(
             AMF_APP_TIMER_INACTIVE_ID) {
       ue_context_p->m5_mobile_reachability_timer.sec =
           (amf_config.nas_config.t3512_min + 4 * 60);
```

Comment:
btw: Is this calculation correct regarding multiplication first?

Reply:
Added additional brackets.